touchin-custom-view/Code/View/view_controller.swift.liquid

```diff
@@ -0,0 +1,56 @@
+import UIKit
+import LeadKit
+import RxSwift
+import RxCocoa
+
+final class {{ module_info.name }}ViewController: BaseCustomViewController<{{ module_info.name }}ViewModel, {{ module_info.name }}View>, {{ module_info.name }}Module, DisposeBagHolder {
+
+    // MARK: - Overrides
+
+    // MARK: - Views
+
+    // MARK: - Properties
+
+    let disposeBag = DisposeBag()
```

Comment:
not sure about rx in view controller
MVP-like arch looks better for me now

Reply:
To my mind all of our current projects use rx in viewController, so I think it's just more convenient)